apps/web/src/lib/site-initializer/index.ts

```diff
@@ -51,12 +52,21 @@ const removeInitializer = () => {
 
 const init = async (store) => {
     /* Init SpaceONE Console */
-    await initConfig();
-    await initApiClient(store, config);
-    const domainName = await initDomain(store, config);
+    try {
+        await initConfig();
+        await initApiClient(store, config);
+        await initDomain(store, config);
```

Comment:
suggestion: I think it would be better to move specific error message to `initDomain()`, and just init route and throw an error at catch block.

Reply:
Thanks,,! I'll do..